active/NumberBox/NumberBox.md

```diff
@@ -0,0 +1,184 @@
+# Number box
+
+<!-- The purpose of this spec is to describe a new feature and
+its APIs that make up a new feature in WinUI. -->
+
+<!-- There are two audiences for the spec. The first are people
+that want to evaluate and give feedback on the API, as part of
+the submission process.  When it's complete
+it will be incorporated into the public documentation at
+docs.microsoft.com (http://docs.microsoft.com/uwp/toolkits/winui/).
+Hopefully we'll be able to copy it mostly verbatim.
+So the second audience is everyone that reads there to learn how
+and why to use this API. -->
+
+## Background
+<!-- Use this section to provide background context for the new API(s) 
+in this spec. -->
+
+<!-- This section and the appendix are the only sections that likely
+do not get copied to docs.microsoft.com; they're just an aid to reading this spec. -->
+
+<!-- If you're modifying an existing API, included a link here to the
+existing page(s) -->
+
+<!-- For example, this section is a place to explain why you're adding this API rather than
+modifying an existing API. -->
+
+<!-- For example, this is a place to provide a brief explanation of some dependent
+area, just explanation enough to understand this new API, rather than telling
+the reader "go read 100 pages of background information posted at ...". -->
+
+
+## Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+
+Number box is a numerics only text control with support for input formatting, validation, basic calculation, and increment stepping.
+
+**Important APIs:** [NumberBox class](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.numberbox) [TODO - generated with MSDN documentation on publication.]
+
+## Is this the right control? 
+
+Use a **NumberBox** control to receive and display formatted numeric values such as currency, IP addresses, and telephone numbers. A **NumberBox** can also be enabled with basic calculator support to compute multiplication, division, addition, and subtraction across parenthetical order with standard operator precedence.
```

Comment:
This still needs updating to reflect the change in scope to not include IP Address, telephone numbers, etc.

Reply:
Correct. I ended the 9-5 having updated the IDL the next draft of APIs and removing vestigial features from the listed examples to avoid any further community investment going into solved questions. This spec received more attention than I think any of us here in Redmond anticipated (see @jevansaks tweets) so my goal tomorrow (today?) is to revise the rest of the spec and its examples to reflect the progress we've made and create some more architect examples. 

Thanks for your patience while I am catching up on all of this **incredible community ambition!**

Reply:
On the note of Twitter, @mrlacey, I will accept your follow, I am just working out a bug with Twitter's help desk as I try and transfer my deprecated accounts username over to my new one. You'll hear from me soon!